test/k8sT/manifests/hubble-cli.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: hubble-cli
+spec:
+  selector:
+    matchLabels:
+      k8s-app: hubble-cli
+  template:
+    metadata:
+      labels:
+        k8s-app: hubble-cli
+    spec:
+      containers:
+        - name: hubble-cli
```

Comment:
I assume the cli is suppose to be together with a hubble server running so we could add affinities to have this DaemonSet only be deployed on systems that have hubble server running more info: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#always-co-located-in-the-same-node